vendor/vendor.json

```diff
@@ -925,6 +931,12 @@
 			"revisionTime": "2016-10-03T13:09:00Z"
 		},
 		{
+			"checksumSHA1": "1quR5pQNYFEgMLp987my6ZXh3vU=",
+			"path": "github.com/igm/sockjs-go/sockjs",
```

Comment:
Thanks for these comments!

- We are (`terminal.go`) only using the gopkg.in version. The github version creeps in due to an import in `gopkg.in/igm/sockjs-go.v2/sockjs/example_handler_test.go` but is unused otherwise.
- Can you (or perhaps, have you already) compiled dashboard with the github sockjs to see if it resolves the problem? If it does I will do some debugging to find out why :)
- I decided to depend on the gopkg.in version to play nice with the release model for sockjs. Unfortunately I haven't done extensive tests _yet_
- The `SockJSURL` problem I've spotted before, I want to serve that file from the dashboard, just haven't got around to do that yet. In my tests the transport that uses that file is not being used so it wasn't a priority. I have added this to the task list now.